### ISSUE TITLE
Error when used with nx@12.6.2

### ISSUE THREAD

**Issue by dan2kx** [Phillip9587/nx-stylelint#110]
I just installed the package and added the configuration to one of my projects and am seeing an error when trying to execute the linting command
```
Cannot find module '@angular-devkit/architect'
Require stack:
- ...root/node_modules/@nrwl/devkit/src/utils/convert-nx-executor.js
- ...root/node_modules/@nrwl/devkit/index.js
- ...root/node_modules/nx-stylelint/src/executors/lint/executor.js
- ...root/node_modules/@nrwl/tao/src/shared/workspace.js
- ...root/node_modules/@nrwl/tao/src/commands/run.js
- ...root/node_modules/@nrwl/tao/index.js
- ...root/node_modules/@nrwl/cli/lib/run-cli.js
```

**Comment by Phillip9587** [comment]
@dan2kx Please show me how you run the linting command. Please also provide the output of `nx report` and your `stylelint` and `nx-stylelint` versions.

**Comment by dan2kx** [comment]
```
  Node : 14.17.3
  OS   : darwin x64
  npm  : 7.20.0
  
  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 12.6.2
  @nrwl/cypress : 12.6.2
  @nrwl/devkit : 12.6.2
  @nrwl/eslint-plugin-nx : 12.6.2
  @nrwl/express : Not Found
  @nrwl/jest : 12.6.2
  @nrwl/linter : 12.6.2
  @nrwl/nest : Not Found
  @nrwl/next : 12.6.2
  @nrwl/node : 12.6.2
  @nrwl/nx-cloud : Not Found
  @nrwl/react : 12.6.2
  @nrwl/schematics : Not Found
  @nrwl/tao : 12.6.2
  @nrwl/web : 12.6.2
  @nrwl/workspace : 12.6.2
  @nrwl/storybook : 12.6.2
  @nrwl/gatsby : Not Found
  typescript : 4.3.5
```

**Comment by Phillip9587** [comment]
@dan2kx Please also provide your stylelint and nx-stylelint version.

**Comment by dan2kx** [comment]
Sorry i missed that

```
"nx-stylelint": "^12.2.0",
"stylelint": "^13.12.0",
```
im running the lint command like so: `nx affected --target=stylelint`

**Comment by remcolakens** [comment]
I had this problem too, just install the missing package seems to fix it for now:

`yarn add -D @angular-devkit/architect`

**Comment by grantglidewell** [comment]
This is a dependency that should resolve within the node package, not the parent. and a very strange dependency to add to a project that doesn't use Angular (only React in our case). If this is truly a peer dependency it should be listed as such, but I don't think it is. 

Ill look into a workaround

**Comment by grantglidewell** [comment]
Digging into this a bit deeper this appears to be an issue stemming from how the Nx package references `@angular-devkit/architect`. This might be resolved when the task to upgrade the Nrwl deps to the current versions is completed.

**Comment by Phillip9587** [comment]
This should be fixed now with version 12.3.0. Please try it out.
